ambry-coordinator/src/test/java/com.github.ambry.coordinator/MockDataNode.java

```diff
@@ -35,36 +32,16 @@
 public class MockDataNode {
   private DataNodeId dataNodeId;
 
-  private Map<BlobId, MaterializedBlob> blobs;
```

Comment:
Nope. I am not complicating things. I went over BlobData and got a clarity that MessageFormat always returns ByteBufferInputStream and hence you have changed the arg in the constructor.